src/test/CMakeLists.txt

```diff
@@ -49,7 +49,7 @@ add_executable(test_bitcoin
   checkqueue_tests.cpp
   coins_tests.cpp
   coinstatsindex_tests.cpp
-  compilerbug_tests.cpp
+  $<$<NOT:$<CXX_COMPILER_ID:MSVC>>:compilerbug_tests.cpp>
```

Comment:
In https://github.com/hebasto/bitcoin/pull/17/commits/de1e9c3da0f9bf0429bf2ef66546c3819af56469: Why is MSVC unable to compile this?

Reply:
[Reworked](https://github.com/hebasto/bitcoin/pull/17#issuecomment-1629206124). Used the same approach as we currently use in `raii_event_tests` and `system_tests`.